nox/command.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, args, env=None, silent=False, path=None,
         self.path = path
         self.success_codes = success_codes or [0]
 
-    def run(self, path_override=None, env_fallback=None):
+    def run(self, path_override=None, env_fallback=None, **kwargs):
```

Comment:
The invocation of `Command` is now sending an additional argument, which is used by `NotifyCommand.__call__`. This means I do not have to have a separate if check for every command subclass (there could eventually be more).